coalib/results/Result.py

```diff
@@ -98,7 +98,7 @@ def from_values(cls,
         :param end_line:        The last related line in the file.
         :param end_column:      The column indicating the last character.
         :param severity:        A RESULT_SEVERITY object.
-        :param debug_msg:       Another message for debugging purposes.
+        :param debug_msg:       Another object useful for debugging purposes by dumping any required data in here.
```

Comment:
As you see, this line is too long !
Take a look at additional_info below for help in doing this correctly